domains/rohith.json

```
@@ -0,0 +1,13 @@
+
+{
+    "description": "Rohith's personal website",
+    "repo": "https://github.com/rohithp7777",
+    "owner": {
+        "username": "rohithp7777",
+        "email": "rohithp8899@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://rohithp7777.github.io",
+        "URL": "https://rohith-1c974.web.app/"
```

Please make sure firebase supports custom domains. If not, you can use github pages.

```suggestion
        "CNAME": "rohith-1c974.web.app"
```